tools/validate_configs/test_configs/hpc-cluster-simple-nfs-sql.yaml

```diff
@@ -28,6 +28,10 @@ deployment_groups:
   - id: network1
     source: modules/network/vpc
 
+  - source: ./community/modules/network/private-service-access
```

Comment:
Please remove the leading `./`.

Reply:
I think I fixed it now, I missed a few on my first pass. Same with the underscores.